packages/playground/data-liberation/project.json

```diff
@@ -33,7 +33,7 @@
 			"options": {
 				"cwd": "packages/playground/data-liberation",
 				"commands": [
-					"./vendor/bin/phpcs --standard=./phpcs.xml -s ./src ./*.php"
+					"./vendor/bin/phpcs --standard=./phpcs.xml -s ./src ./*.php "
```

Comment:
Quick question. Why add a space here? To ease the CLI script concatenating?

Reply:
ah that's just a typo, good catch

Reply:
Ok cool. I was just curious if I was missing something 🙂